_summaries/zhao2020point_2.md

```diff
@@ -1,19 +1,68 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
----
+title: Point Transformer
+paper: Zhao2020point
+author: # sritank
+score: # 9/10
+
+* Core idea?
+
+They design a Point Transformer layer for point cloud processing suitable for point cloud labeling and segmentation. They introduce vector self-attention and trainable position encoding among others to make their residual transformer network invariant to permutation and cardinality. 
+
```

Comment:
The paper mentions a few times that the transformer and self-attention concept is apt for point cloud tasks because the self-attention operator is essentially a set operator and the 3D points are embedded sets. This point could provide some context as to why they considered transformers.